vue-frontend/src/components/home-new/ContributionSection.vue

```diff
@@ -1,27 +1,27 @@
 <template>
-  <div class="tw-relative">
-    <div class="tw-container tw-mt-2 md:tw-mt-4">
-      <div
-        class="tw-sticky tw-w-full tw-h-[30vh] sm:tw-h-[25vh] lg:tw-h-[35vh] tw-z-[1] tw-top-[6rem] tw-bg-white tw-pt-12 tw-text-center"
+  <div class="tw-container tw-!py-12 tw-text-center">
+    <h1
+      class="tw-font-inter-bold tw-text-black-core/[.87] tw-text-[1.875rem] tw-leading-[2.5rem] md:tw-text-[3.4375rem] md:tw-leading-[5.5625rem]"
+    >
+      We are here to Contribute
+    </h1>
+    <div class="tw-w-[290px] sm:tw-w-full tw-mx-auto tw-pb-14 md:tw-pb-20">
+      <span
+        class="tw-font-inter-medium tw-text-black-core/[0.6] tw-text-[1rem] tw-leading-[1.375rem] md:tw-text-[1.5rem] md:tw-leading-[2.5625rem]"
+        >and that starts with the site itself,
+        <a
+          class="tw-relative tw-pb-[5px] v2-canopas-gradient-text tw-font-inter-medium gradient-underline tw-no-underline after:tw-absolute after:tw-w-full after:tw-h-[2px] after:tw-bg-gradient-to-l tw-to-pink-300 tw-from-orange-300 tw-bg-gradient-[1deg] after:tw-bottom-0 after:tw-left-0"
```

Comment:
Let's put bg properties before to padding and text properties

Reply:
As its a pseudo property i have kept it in last.` tw-bg-gradient-[1deg]` you told me before to put it here with ` after:tw-bg-gradient-to-l` .